src/main/java/com/jfrog/ide/idea/ui/JFrogCiToolWindow.java

```diff
@@ -49,33 +57,72 @@ public class JFrogCiToolWindow extends AbstractJFrogToolWindow {
     private LinkButton seeMore;
     private JLabel branch;
     private JLabel commit;
+    private final OnePixelSplitter rightVerticalSplit;
+    final CiComponentsTree componentsTree;
+    ComponentIssuesTable issuesTable;
+    JScrollPane issuesDetailsScroll;
+    JPanel moreInfoPanel;
 
     public JFrogCiToolWindow(@NotNull Project project, boolean buildsConfigured) {
-        super(project, buildsConfigured, CiComponentsTree.getInstance(project));
+        super(project);
+        this.componentsTree = CiComponentsTree.getInstance(project);
+        JPanel toolbar = createActionToolbar();
+
+        JComponent issuesPanel = createComponentsIssueDetailView();
+
+        OnePixelSplitter leftVerticalSplit = new OnePixelSplitter(false, 0.5f);
+        leftVerticalSplit.setFirstComponent(createComponentsTreeView());
+        leftVerticalSplit.setSecondComponent(issuesPanel);
+
+        rightVerticalSplit = new OnePixelSplitter(false, 0.6f);
+        rightVerticalSplit.setVisible(false);
+        rightVerticalSplit.setFirstComponent(leftVerticalSplit);
+        rightVerticalSplit.setSecondComponent(createMoreInfoView(buildsConfigured));
+
+        setToolbar(toolbar);
+        setContent(rightVerticalSplit);
+        registerListeners();
     }
 
-    @Override
-    String getComponentsTreeTitle() {
+    private String getComponentsTreeTitle() {
         return " Build Components (Issues #)";
     }
 
-    @Override
+    /**
+     * Create CI or local issues filter menu
```

Comment:
Same comment for the other methods in this file
```suggestion
     * Create CI issues filter menu
```